pkg/frontend/frontend_scheduler_worker.go

```diff
@@ -235,20 +237,24 @@ type frontendSchedulerWorker struct {
 	// Number of queries sent to this scheduler.
 	enqueuedRequests prometheus.Counter
 
+	// Number of queries that failed to forward to the querier.
+	forwardedRequestsFailedStats *expvar.Int
```

Comment:
I'm wondering if we should use usagestats for this. Would a `prometheus.Counter` be more suitable?

Reply:
Yes I agree. We use usage stats for the "phone home" functionality and wouldn't collect operational metrics to that detail there.